packages/@aws-cdk/aws-cloudwatch/test/test.alarm.ts

```diff
@@ -17,13 +17,15 @@ export = {
     new Alarm(stack, 'Alarm', {
       metric: testMetric,
       threshold: 1000,
-      evaluationPeriods: 2
+      evaluationPeriods: 3,
+      datapointsToAlarm: 2,
```

Comment:
It's usually a better practice to add a test instead of modifying an existing test. This way you cover a bit more of the surface area (with and without this option).

Reply:
Uh yeah sure.